### PR TITLE
add other services to cypress config

### LINKED ISSUE #2692: Add all services to cypress/config/services.js
**Is your feature request related to a problem? Please describe.**
We are missing a lot of services in this file. Add all that are in the app config folder (except default)

**Describe the solution you'd like**
add:
`  serviceName: {
    font: undefined,
    pageTypes: {
      articles: undefined,
      frontPage: undefined,
    },
  },`
for every missing service

**Describe alternatives you've considered**
n/a

**Testing notes**
[Tester to complete]
No testing needed

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #2692 

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -1,11 +1,99 @@
...
+    pageTypes: { articles: undefined, frontPage: undefined },
+  },
   igbo: {
     font: undefined,
     pageTypes: {
       articles: undefined,
       frontPage: '/igbo',
     },
   },
+  index: {
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]: not needed

File: cypress/support/config/services.js [hunk 2]
```diff
@@ -1,11 +1,99 @@
...
+  },
+  korean: {
+    font: undefined,
+    pageTypes: { articles: undefined, frontPage: undefined },
+  },
+  kyrgyz: {
+    font: undefined,
+    pageTypes: { articles: undefined, frontPage: undefined },
+  },
+  loadableConfig: {
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]: not needed

File: cypress/support/config/services.js [hunk 3]
```diff
@@ -35,11 +122,83 @@ export default {
...
+  },
+  ukranian: {
+    font: undefined,
+    pageTypes: { articles: undefined, frontPage: undefined },
+  },
+  urdu: {
+    font: undefined,
+    pageTypes: { articles: undefined, frontPage: undefined },
+  },
+  uzbec: {
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]:
```suggestion
  uzbek: {
```

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Just one typo

**Review by jamesbhobbs — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing.

**Pull request merged**